### ISSUE TITLE
Quests plugin not working or responding

### ISSUE THREAD

**Issue by GamerDude3000** [PikaMug/Quests#1321]
**Quests Version/Build:** Quests-3.9.4

**What is the issue?:** When I install the latest version of Quests, I simply put it in the plugins folder and load up the server. It says that the plugin is loaded and working fine when I do /plugins, but whenever I go to use any commands for the plugin, nothing happens. Nothing occurs in the console and none of the commands work. It simply acts as if nothing is happening. No console response or in-game response. I am running a 1.13.2 server from a home PC host if that is worth any information. Is there something I am doing wrong? What could possibly fix this? Let me know if you need me to provide any more information as well.

**Comment by PikaMug** [comment]
Thanks for the report. Sounds like a permissions issue - try elevating yourself to OP and/or give the latest development build a try at https://ci.codemc.io/job/PikaMug/job/Quests/126/

**Comment by GamerDude3000** [comment]
So I tried running the current development build (Build 3.9.5) and I seem to still have the same issue. I have been OP'ed on both versions 3.9.5 and 3.9.4 and nothing is still happening. I decided to make a quick little video demonstrating what happens, as well as show that the plugin is in fact loaded in the server files.
![PluginIsLoaded](https://user-images.githubusercontent.com/25123459/84070428-bfb6c600-a991-11ea-8693-a8a5d6da3eb9.png)
![Quests Not Working Demo](https://user-images.githubusercontent.com/25123459/84070709-2f2cb580-a992-11ea-8439-ec6c1750e816.gif)

**Comment by PikaMug** [comment]
Thanks for that. I'm struggling to recreate this on my 1.13.2 test server. Another possibility is that another plugin is cancelling the commend event. Would you please try removing all non-essential plugins for a moment? If that resolves the issue, add them back one-by-one until you find the conflict.

Also, which permissions plugin are you using?

**Comment by GamerDude3000** [comment]
I am using PermissionsEX (PEX) soon to be transferred to LuckPerms. After creating a separate test server, one by one, I added each plugin with the quests 3.9.4 plugin and to my surprise, none of the plugins were causing it to not work. The quests plugin worked when I added every plugin. So I ended up just copying all of my server plugins, and deleting all 51 one in the plugins folder, and then pasting the jar files back into the plugin folder. And surprisingly it worked and the plugin is working now. I have no clue why this was the case but now it works so I'm fine with it. I appreciate your help!
